app/src/pages/explorer/curate/spreadsheet/spreadsheet-upload-script.js

```diff
@@ -0,0 +1,195 @@
+import DropZone from '@/components/curate/FileDrop.vue'
+import FilePreview from '@/components/curate/FilePreview.vue'
+import LoginRequired from '@/components/LoginRequired.vue'
+import Dialog from '@/components/Dialog.vue'
+import CurateNavBar from '@/components/curate/CurateNavBar.vue'
+import Spinner from '@/components/Spinner.vue'
+import useFileList from '@/modules/file-list'
+import { VERIFY_AUTH_QUERY, USER_DATASET_IDS_QUERY, CREATE_DATASET_MUTATION } from '@/modules/gql/dataset-gql'
+import { mapGetters, mapMutations } from 'vuex'
+
+// Create separate file objects for spreadsheet vs supplementary files
+const spreadsheetFn = useFileList()
+const suppFn = useFileList()
+
+export default {
+  name: 'SpreadsheetHome',
+  components: {
+    dialogbox: Dialog,
+    DropZone,
+    FilePreview,
+    CurateNavBar,
+    Spinner,
+    LoginReq: LoginRequired
+  },
+  data () {
+    return {
+      auth: true,
+      verifyUser: null,
+      loading: false,
+      uploadInProgress: true,
+      selectedDataset: {
+        label: '',
+        id: null
+      },
+      spreadsheetFiles: spreadsheetFn.files,
+      suppFiles: suppFn.files,
```

Comment:
Should this two be computed properties instead, and instead of calling those functions, you return them? Does suppFn require initialization? Let me know what your thoughts are